packages/petit_parser/src/parsers/miscellaneous/any.candy

```diff
@@ -0,0 +1,20 @@
+use ....parsers
+
+public fun any<Item>(message: String = "Expected any input."): Parser<Item, Item> {
+  AnyParser<Item>(message)
+}
+
+class AnyParser<Item> {
+  let message: String
+}
+impl<Item> AnyParser<Item>: Parser<Item, Item> {
+  fun parse(context: Context<Item>): Result<Success<Item>, Failure> {
+    let buffer = (context.buffer as Iterable<Item>)
```

Comment:
```suggestion
    let buffer = context.buffer as Iterable<Item>
```

Reply:
Unfortunately, our old parser has problems without the parentheses…